ftp.go

```diff
@@ -184,6 +184,10 @@ func (c *ServerConn) setUTF8() error {
 	}
 
 	code, message, err := c.cmd(-1, "OPTS UTF8 ON")
+	// Workaround for FTP servers, that does not support this option.
+	if code == StatusBadArguments {
```

Comment:
I think this block should be after the next block, `if err != nil` since `c.cmd()` seems to return `0` as the code on error.

Reply:
You are right, done.